proto/qdrant/cloud/account/v1/account.proto

```diff
@@ -299,6 +304,11 @@ message CreateAccountInviteRequest {
   // The details of the invite to create.
   // This is a required field.
   AccountInvite account_invite = 1 [(buf.validate.field).required = true];
+  option (buf.validate.message).cel = {
```

Comment:
this kind of validations are going to be hard to maintain because every time we add a new read-only field we would need to remember extending this list. I don't have an alternative, but it would be great to find one.